url/javascript-urls.window.js

```diff
@@ -0,0 +1,49 @@
+// The results of this test are all over the map due to browsers behaving very differently for
+// javascript: URLs.
+//
+// Chromium is pretty close execution-wise, but it parses javascript: URLs incorrectly.
+// Gecko navigates to non-string return values of the result of executing a javascript: URL.
+// WebKit executes javascript: URLs too early and has a harness error due to URL parsing.
+//
+// The expectations below should match the HTML and URL standards.
+[
+  {
+    "description": "javascript: URL that fails to parse due to invalid host",
+    "input": "javascript://test:test/%0aglobalThis.shouldNotExistA=1",
+    "property": "shouldNotExistA",
+    "expected": undefined
+  },
+  {
+    "description": "javascript: URL that fails to parse due to invalid host and has a U+0009 in scheme",
+    "input": "java\x09script://test:test/%0aglobalThis.shouldNotExistB=1",
+    "property": "shouldNotExistB",
+    "expected": undefined
+  },
+  {
+    "description": "javascript: URL without an opaque path",
+    "input": "javascript://host/1%0a//../0/;globalThis.shouldBeOne=1;/%0aglobalThis.shouldBeOne=2;/..///",
+    "property": "shouldBeOne",
+    "expected": 1
+  }
```

Comment:
Maybe something like

```json
"javascript:globalThis.shouldBeAString = \"https://whatsoever.com/?a=b&c=5&x=y\""
```
per the OP of https://github.com/whatwg/url/issues/307 ?